core/src/main/java/hudson/init/InitMilestone.java

```diff
@@ -82,7 +82,7 @@
      * By this milestone, all programmatically constructed extension point implementations
      * should be added.
      */
-    EXTENSIONS_AUGMENTED("Augmented all extensions"),
+    EXTENSIONS_AUGMENTED("Augmented all extensions"), // TODO nothing attains() this so when does it actually happen?
```

Comment:
Unrelated to the PR. 
There is implicit ordering of this enum in Jenkins startup, hence it will happen between PLUGINS_STARTED and the beginning of the Job loading phase